kong/tracing/instrumentation.lua

```diff
@@ -72,33 +72,32 @@ function _M.balancer(ctx)
     local try = balancer_tries[i]
     span = tracer.start_span("balancer try #" .. i, {
       kind = 3, -- client
-      start_time_ns = try.balancer_start * 100000,
+      start_time_ns = try.balancer_start * 1000000,
```

Comment:
We should define a constant, such as `xxx = 1000* 1000`.

Reply:
It's a more proper way to achieve that, thanks!